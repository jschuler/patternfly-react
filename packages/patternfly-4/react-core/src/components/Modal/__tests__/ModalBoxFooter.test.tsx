// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { ModalBoxFooter } from '../ModalBoxFooter';

describe('ModalBoxFooter test', () => {
  it('ModalBoxFooter should match snapshot', () => {
    const view = shallow(
      <ModalBoxFooter
        children={<>ReactNode</>}
				className={"''"}
				isLeftAligned={false}
      />);
    expect(view).toMatchSnapshot();
  });
});
