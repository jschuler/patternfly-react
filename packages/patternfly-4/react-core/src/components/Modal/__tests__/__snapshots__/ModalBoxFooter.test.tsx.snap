// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalBoxFooter test ModalBoxFooter should match snapshot 1`] = `
<div
  className="pf-c-modal-box__footer ''"
>
  ReactNode
</div>
`;
